server/py/services/api/tests/unit/crud/test_alerts.py

```diff
@@ -315,38 +313,3 @@ async def test_alert_reset_with_fields_updates(
         else alert_objects.AlertActiveState.ACTIVE
     )
     assert alert.state == expected_state
-
-
-def _generate_alert_data(
-    project,
-    name,
-    entity,
-    summary,
-    trigger,
-    description=None,
-    severity=alert_objects.AlertSeverity.LOW,
-    notifications=None,
-    criteria=None,
-    reset_policy=alert_objects.ResetPolicy.AUTO,
-):
-    if notifications is None:
-        notification = mlrun.common.schemas.Notification(
-            kind="slack",
-            name="slack_notification",
-            secret_params={
-                "webhook": "https://hooks.slack.com/services/",
-            },
-        )
-        notifications = [alert_objects.AlertNotification(notification=notification)]
-    return alert_objects.AlertConfig(
-        project=project,
-        name=name,
-        description=description,
-        summary=summary,
-        severity=severity,
-        entities=entity,
-        trigger=trigger,
-        criteria=criteria,
-        notifications=notifications,
-        reset_policy=reset_policy,
-    )
```

Comment:
Do we still need to move this?

Reply:
yes, we can use it in our upcoming work when adding alert activations tests to crud ( that mocks the db )